File: Q8/Q8.cpp
#include<bits/stdc++.h>
using namespace std;

string arrangedString(string f,string s,string t){
    for(int i=0;i<f.length();i++){
        if (f[i] == 'a' || f[i] == 'e' || f[i] == 'i' || f[i] == 'o' || f[i] == 'u' ){
            f[i]='%';
        }
    }
    for(int i=0;i<s.length();i++){
        if (s[i] == 'a' || s[i] == 'e' || s[i] == 'i' || s[i] == 'o' || s[i] == 'u'){
            continue;
        }
        else{
            s[i]='#';
        }
    }
    transform(t.begin(),t.end(),t.begin(),::toupper);
    return f+s+t;
}
int main(){
    string first ,second,third;

    cin>>first>>second>>third;

    string NewString=arrangedString(first,second,third);

    cout<<NewString;
}

File: Q8/Q8.py
first=input("Enter the first string : ")
second=input("Enter the second string : ")
third =input("Enter the third string : ")

firstString=list(first)
secondString=list(second)
for i in range(len(firstString)):
    if firstString[i] in 'aeiouAEIOU' :
        firstString[i]="%"
    else :
        continue

for i in range(len(secondString)):
    if secondString[i] in 'aeiouAEIOU':
        continue
    else:
        secondString[i]="#"

first=''.join(firstString)
second=''.join(secondString)
third=third.upper()

print(first+second+third)